### DISCUSSION TITLE
How can I customise the color for the text following "Search instead for"

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [benbusby/whoogle-search#1039] [Q&A] [answered]
I'm using a self-hosted instance of Whoogle with Catppuccin colors. The text following "Search instead for" is a dark blue that is difficult to read against the background.

![image](https://github.com/benbusby/whoogle-search/assets/20235646/251d8fee-0554-4c10-bf70-416eb859a1b3)

Is it possible to style that text separately? None of the custom themes I looked at seemed to be doing that so I'm not sure if that's presently possible.

**Comment by binitGajera** [comment, marked as answer]
`#scc div:nth-child(2) a {color: red !important}}`

Changing the `color: red` above to whatever color you need should do the trick.
